src/Components/Book.js

```diff
@@ -1,27 +1,16 @@
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ title, author, onDelete }) => {
-  const [isDeleting, setIsDeleting] = useState(false);
-
-  const handleDelete = () => {
-    setIsDeleting(true);
-    // Call your delete book API or perform any other delete action
-    onDelete();
-  };
-
-  return (
-    <div>
-      <h3>{title}</h3>
-      <p>
-        Author:
-        {author}
-      </p>
-      {!isDeleting && <button type="button" onClick={handleDelete}>Delete</button>}
-      {isDeleting && <p>Deleting...</p>}
-    </div>
-  );
-};
+const Book = ({ title, author, onDelete }) => (
+  <div>
+    <h3>{title}</h3>
+    <p>
+      Author:
+      {author}
+    </p>
+    <button type="button" onClick={onDelete}>Delete</button>
+  </div>
+);
```

Comment:
- [ ] Thank you for your work on this feature! However, as part of the project requirements, it's important that the button for removing a book is implemented as an independent `<Button>` component. This component should have an event handler that removes the book from the books array by ID. Currently, I noticed that you're using the default HTML `<button>` element instead of the required <Button> component so please make sure to update this.
- [ ] This also applies to the button for adding a book in the `BookForm` component, on that component I noticed you're handling the submit from the form `onSubmit` so you'll have to update that as well. ✔